compliance/aws/rds_eos_instances/aws_eos_rds_instances.pt

```diff
@@ -0,0 +1,828 @@
+name "AWS RDS instances with support ending soon"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report on AWS RDS Instances that are about to end support. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/aws/rds_eos_instances) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."

```

Comment:
For clarity, I'd recommend changing the description to something like this to clearly communicate the value prop of using this:

Report on AWS RDS Instances that are about to transition from standard support to more expensive extended support.